_docs/concepts/fault-injection.md

```diff
@@ -0,0 +1,43 @@
+---
+category: Concepts
+title: Fault Injection
+
+parent: Traffic Management
+order: 40
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+ 
+While Envoy sidecar/proxy provides a host of
+[failure recovery mechanisms](./handling-failures.html) to services running
+on Istio, it is still
+imperative to test the end-to-end failure recovery capability of the
+application as a whole. Misconfigured failure recovery policies (e.g.,
+incompatible/restrictive timeouts across service calls) could result in
+continued unavailability of critical services in the application, resulting
+in poor user experience.
+
+**Systematic fault injection:** Istio advocates a systematic approach to
+testing the failure recovery of the application as a whole as opposed to a
+chaotic model. Specifically, Istio implements fault injection at the Envoy
```

Comment:
Specifically, Istio enables protocol-specific fault injection into the network, instead of killing pods...